### PR TITLE
secondary menu tab first commit

### PR DESCRIPTION
**Pull request by tiffy74**
### Prerequisites

If there's an existing issue for this PR then this fixes [Issue 5277 - no.95 and no.100](https://github.com/umbraco/Umbraco-CMS/issues/5277)

### Description

Added role=tablist to ul of dashboard menu and added role=tab to li items of dashboard menu.  This means they will now not be skipped by navigation

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<ul class="umb-tabs-nav">
-    <li class="umb-tab" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
+<ul role="tablist" class="umb-tabs-nav">
+    <li class="umb-tab" role="tab" aria-selected="true" aria-controls="panel-1" id="tab-1" tabindex="0" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
```
Comment (bjarnef) [inline src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html]: @tiffy74 it seems to be invalid, where all tabs get the id `tab-1`. You could change it to `id="tab-{{$index + 1}}"` and maybe `tabindex="{{$index}}"`.

Comment (tiffy74) [reply src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html]: Thanks for the response

Actually I had intended it to be simpler - I just forgot to remove the additional elements from my own testing code.  The id="tab-1" was meant to be removed.

It should be:

`<li class="umb-tab" role="tab" aria-selected="true" tabindex="0" ng-repeat="tab in vm.tabs |  limitTo: vm.maxTabs" data-element="tab-{{tab.alias}}" ng-class="{'umb-tab--active': tab.active, 'umb-tab--error': tabHasError}" val-tab>
        <a ng-href="" ng-click="vm.clickTab($event, tab)">{{ tab.label }}</a>
    </li>`

Comment (tiffy74) [reply src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html]: Have now updated the branch and removed the id and aria-controls elements - tested and works as expected.

**Comment by nul800sebastiaan** [comment]
Hi @tiffy74  - before I review this, the title of this PR is a bit ambiguous as to whether the work on this PR has completed or not? 

I see there's a merge conflict but that's fine, something I can resolve easily when reviewing 👍 

So let me know if you're still working on this PR or if it's ready for review.

**Comment by tiffy74** [comment]
Hiya @nul800sebastiaan 

It is ready for review.  Think the merge conflict is the package-lock file so as you say can be resolved easily.

Ive removed the items that @bjarnef was highlighting and so should work fine now

Thanks

Tiff

**Comment by nul800sebastiaan** [comment]
Thanks @tiffy74 ! I've reverted the package-lock and am happy with the proposed change. Thanks very much! 🎖

**Pull request merged**